Example/IntegrationTests/Push/PushTests.swift

```diff
@@ -91,55 +91,40 @@ final class PushTests: XCTestCase {
         makeWalletClients()
     }
 
-    func testRequestPush() async {
```

Comment:
push request step removed from protocol right?

Reply:
exactly, the method is deprecated